### ISSUE TITLE
Secondary nav at bottom for accessibility

### ISSUE THREAD

**Issue by bjorvack** [CodingTrain/website#840]
On pages with a lot of links users need to tab a lot. A secondary nav in the footer or a back to menu hidden link could help visually impaired users to return to the menu

**Comment by phucduong86** [comment]
Hi @bjorvack , 
As screen reader user, a user can pull up the regions and directly navigate to the Navigation. I've recently updated the code so that it is semantically shown as nav region and most of the AT should be able to pick it up:
![image](https://user-images.githubusercontent.com/21692768/58716588-62a82d80-8397-11e9-8049-c745263c26f7.png)

For user who tab through the page it is actually not ideal since they have to tab through ALL the element to get to the bottom navigation. For usability a floating button that allows user to scroll to the top of the page is great for long pages, it would benefits sighted user though and not the audience you are thinking off.
